weblate/billing/models.py

```diff
@@ -193,7 +193,7 @@ class Billing(models.Model):
     # with payment processor
     payment = models.JSONField(editable=False, default=dict, encoder=DjangoJSONEncoder)
 
-    objects = BillingManager.from_queryset(BillingQuerySet)()
+    objects: BillingQuerySet = BillingManager.from_queryset(BillingQuerySet)()
```

Comment:
Oh, I'd overlooked this one. I've tried to investigate a bit but couldn't find a working solution, just this issue ticket typeddjango/django-stubs#1067 or `#type: ignore[assignment]` or using `cast` (which seems too brutal)

Should I remove this change?

Reply:
Probably yes. I think this is best to be addressed at django-stubs or mypy, otherwise we end up doing that on every model.

Reply:
Furthermore, this does address only accessing query set via `.objects` but not when using related manager.

Reply:
That's true. Thanks :smirk: